aiida/cmdline/params/options/main.py

```diff
@@ -90,18 +90,19 @@ def decorator(command):
     return decorator
 
 
-def set_log_level(ctx, __, value):
-    """Set the log level for the global ``AIIDA_LOGGER``.
+def set_log_level(_, __, value):
```

Comment:
just to understand: the first to arguments were arguments that became obsolete over time?

Perhaps it would be worth mentioning this in the docstring.

Reply:
This is the standard interface for a `click` callback. The first is the `ctx` context and the second `param` is the parameter. We used to use the `ctx` to get the loaded profile, but now it is fully decoupled so we no longer need it.

Reply:
> This is the standard interface for a `click` callback.

I see, perhaps good to mention this in the docstring?
I would have tended to use `_ctx` and `_param` as the argument names to make it easier for the next guy who has to start using them again.

Reply:
Done